internal/kubernetes/pvc.go

```diff
@@ -55,7 +55,10 @@ func ListAllPVCWithStorageclass(client *k8s.Clientset, scName string) (*[]corev1
 					sc = &val
 				}
 			}
-			if *p.Spec.StorageClassName == scName {
+			if p.Name == pvcName {
+				*pl = append(*pl, p)
```

Comment:
This will get the first PVC with that name, from any namespace. To convert a specific PVC, we really should take the namespace as a param as well so it's guaranteed which PVC is being converted. We wouldn't want to convert a PVC of the same name in the wrong namespace.

Reply:
how about checking PVC name in storage class as we are already taking storage class and we are checking storage class for other scenario too

Reply:
It's the namespace of the app pod, not the namespace of the rook cluster, right? So we would need the app namespace as a param.

Reply:
Maybe I'm missing something but why do we need to consider the namespace of the pod(application pod will be down during the migration) and checking PVC per storage class should give the right PVC to migrate. Let's discuss more

Reply:
Since the scenario is to convert a single pvc, the pvc can be identified exactly by its name and namespace, then it can be retrieved directly. We shouldn't need to iterate through all the namespaces, since there might be another pvc with the same name in a different namespace. We don't want to convert the wrong pvc in a different namespace.